gflownet/gflownet.py

```diff
@@ -434,14 +449,39 @@ def sample_batch(
                         is_forward=True,
                         temperature=self.temperature_logits,
                         random_action_prob=self.random_action_prob,
+                        mask_invalid_actions=masks_invalid_actions_forward,
```

Comment:
Idem. I would simply pass `None` and remove the definition of `masks_invalid_actions_forward = None` in line 428 above.